docker/data_source_docker_plugin.go

```diff
@@ -0,0 +1,75 @@
+package docker
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+func dataSourceDockerPlugin() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceDockerPluginRead,
+
+		Schema: map[string]*schema.Schema{
+			"id": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"alias": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: "Docker Plugin alias.",
+			},
+
+			"plugin_reference": {
```

Comment:
all but id and alias need to be Computed: true.
id and alias should conflict

Reply:
I want to know what is a problem if `Computed: true` isn't set in Data Sources.

https://www.terraform.io/docs/extend/schemas/schema-behaviors.html#computed

Reply:
https://github.com/hashicorp/terraform-plugin-sdk/blob/aa07c4354f9a8fd948ed40f020f340132ada44bc/helper/schema/schema.go#L78-L79

Reply:
Oh, I misunderstood.
I'll fix. Thanks.

Reply:
Fixed. c8b9c7fc8c21c4a40def5e3c0d21ede4b0d87db3